build/Targets/VSL.Imports.targets

```diff
@@ -391,73 +394,55 @@
 
        ==================================================================================== -->
 
-  <UsingTask TaskName="SignFiles" AssemblyFile="$(TF_BUILD_BUILDDIRECTORY)\MicroBuild\MicroBuild.Signing.dll" Condition="'$(RunningInMicroBuild)' == 'true'" />
+  <UsingTask TaskName="SignFiles" AssemblyFile="$(MicroBuildSigningLibrary)" Condition="'$(ShouldSignBuild)' == 'true'"/>
 
+  <!-- The below properties are set so that the SignVsixInputs Target runs right after we Get the VSIX Source items
+       This allows the signing of the components that go into the VSIX, which gets signed through SignVsix Target-->
   <PropertyGroup>
     <CreateVsixContainerDependsOn>$(CreateVsixContainerDependsOn);SignVsixInputs</CreateVsixContainerDependsOn>
     <PrepareForRunDependsOn>$(PrepareForRunDependsOn);SignVsix</PrepareForRunDependsOn>
-    <ProducingSignedVsix Condition="'$(ShouldSignBuild)' == 'true' AND '$(NonShipping)' != 'true' AND '$(CreateVsixContainer)' == 'true'">true</ProducingSignedVsix>
```

Comment:
We have one VSIX and it is signed. I dont know if we are going to have un-signed VSIX. If we do in the future then we can modify this information. I dont want to overload information where we have a simple scenario to deal with.